ultra/tests/task/test_multiagent_mock_2lane_t_70kmh_p-test-flow-1/map.net.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
(applies to all xml files, glb, pkl?)

Reply:
I think for now it would be best to keep them. They are not generated before certain CI tests and thus must be shipped with ULTRA (see the current scenarios used for single agent CI tests: https://github.com/huawei-noah/SMARTS/tree/master/ultra/tests/task). Perhaps we can change the tests in the future to generate these on the fly so that they do not have to be shipped with ULTRA.